src/options/options.ts

```diff
@@ -7,16 +7,15 @@ export interface TsAutoMockOptions {
   cacheBetweenTests: TsAutoMockCacheOptions;
 }
 
-let options: TsAutoMockOptions = null;
+let tsAutoMockOptions: TsAutoMockOptions = defaultOptions;
 
-export function SetTsAutoMockOptions(_options: TsAutoMockOptions): void {
-  options = _options;
+export function SetTsAutoMockOptions(options?: TsAutoMockOptions): void {
```

Comment:
You could remove the question mark and do an if statement in the transformer. if(options) { SetTsAUtoMockOptions(options)}

Reply:
That makes sense and I agree, it should be the user's responsibility to pass in a truthy value.